README.md

```diff
@@ -112,4 +134,5 @@ google is up
 
 ## License
 
-MIT &copy; [**Travis Clarke**](https://blog.travismclarke.com/)
+MIT &copy; [**Travis Clarke**](https://blog.travismclarke.com/),
+           [Sebastian Pipping](https://blog.hartwork.org/)
```

Comment:
Feel free to add your name to the [LICENSE](https://github.com/clarketm/wait-for-it/blob/master/LICENSE) as well! Also, I sent you a collaborator invite.

Reply:
Very kind, thank you! :pray: I have added myself to `LICENSE` now and I gladly accept the invitation.